src/components/dialogs/contingency-list/modification/contingency-list-modification-dialog.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useSelector } from 'react-redux';
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import {
+    editContingencyList,
+    getEmptyFormData,
+    getFormDataFromFetchedElement,
+    getSchema,
+} from '../contingency-list-utils';
+import { useEffect, useMemo } from 'react';
+import { fetchContingencyList } from '../../../../utils/rest-api';
+import CustomMuiDialog from '../../custom-mui-dialog';
+import ContingencyListModificationForm from './contingency-list-modification-form';
+import { NAME } from '../../../utils/field-constants';
+
+const emptyFormData = getEmptyFormData();
+
+const ContingencyListModificationDialog = ({
```

Comment:
I don''t think this component should exist.
In GridStudy, we have creation and modification components, because it's 2 different kind of network modification.
If you want to edit a nework modification "creation", we open the creation component.
For me it should be the same here.

Reply:
Yes, that what i thought also (that's what i did at first). but i think seperate the two component make them easier to understand and seperate the fetch and requests sent  by each componenet ... and the two dialog use the same componenets

Reply:
Ok I understand, but it's confusing because in GridStudy it's not like that. Mayve at least rename "Modification" by "Edit" instead ?